src/loaders/svg/tvgSvgUtil.cpp

```diff
@@ -163,14 +162,21 @@ float svgUtilStrtof(const char *nPtr, char **endPtr)
         unsigned int expo_part;
         int minus_e;
 
-        iter++;
+        ++iter;
+
+        //Exception: svg may have 'em' unit for fonts. ex) 5em, 10.5em
+        if ((*iter == 'm') || (*iter == 'M')) {
+            //TODO: We don't support font em unit now, but has to multiply val * font size later...
+            a = iter + 1;
+            goto on_success;
+        }
 
         //signed or not
         minus_e = 1;
         if (*iter == '-')
         {
             minus_e = -1;
-            iter++;
+            ++iter;
```

Comment:
out of curiosity - is there reason for which you have changed it? there are still other places where 'iter++' is used

Reply:
for consistency. you can change it others.

Reply:
@mgrudzinska I'm not just hang around to change the coding convention, when touch the code by the chance then it's the time to change violations near my code.

Reply:
ok, sure, I understand and I'll remember to use ++x